src/components/sets/TreeNode.js

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/no-autofocus */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
I agree. The only one that I do not foresee being able to remove is the `no-autofocus` one since it has been discussed that the input needs to be focused immediately after selecting a set to reduce keystrokes.